src/controllers/auth/index.js

```diff
@@ -0,0 +1,83 @@
+const { User } = require("../../models");
+
+const login = async (req, res) => {
+  try {
+    const { email, password } = req.body;
+    const user = await User.findOne({ where: { email } });
+    if (user) {
```

Comment:
More nested `if/else` - rule of thumb, check the unhappy/error path with an `if` first and then everythign else can go in the normal flow of code.